WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -1,14 +1,46 @@
 import XCTest
+@testable import struct Yosemite.POSProduct
 @testable import WooCommerce
 
 final class PointOfSaleDashboardViewModelTests: XCTestCase {
 
+    private var sut: PointOfSaleDashboardViewModel!
+    private var cardPresentPaymentService: CardPresentPaymentPreviewService!
```

Comment:
Is `CardPresentPaymentPreviewService` something we'll use for both previews and as a mock? Or we plan to separate these? If we don't maybe we should rename this to `CardPresentPaymentMockService` or similar 🤔

Reply:
maybe @joshheald knows a bit more?

Reply:
I only added it to make it easier to do previews. It's the simplest possible implementation of `CardPresentPaymentFacade`, and importantly has no dependencies.

It kinda works as a test double as well, but it's not really got some of the stuff we're likely to want when testing, e.g. spy variables, the opportunity to set test state/mocked responses.

Adding those would make it worse at its primary job (being a simple structure to make preview creation easy) so we wouldn't want to add that, or rename it to make the role of the class less clear.

Adding a new mock is the way to go here, IMO.